### PR TITLE
Initial Version of Offline Actions

### PR DESCRIPTION
**Pull request by manuelroemer**
## Requirements
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

The initial version of the offline actions. When comparing with the designs, there are still several differences to be done, but this is a good intermediary state.

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Review by FlorianRappl — APPROVED** [review]
LGTM - thanks! :beers:

**Comment by brandones** [comment]
Hey @manuelroemer , what's the status of this ticket?

**Comment by manuelroemer** [comment]
Oh right, this is still open, I forgot about it. I'll have to fix the build and then we can merge from my side.

File: packages/apps/esm-implementer-tools-app/src/configuration/editable-value.component.tsx [hunk 1]
```diff
@@ -136,7 +136,7 @@ export default function EditableValue({
                 hasIconOnly
                 onClick={() => {
                   temporaryConfigStore.setState(
-                    unset(temporaryConfigStore.getState(), ["config", ...path])
+                    unset(temporaryConfigStore.getState(), ["config", ...path]) as any
```
Comment (manuelroemer) [inline packages/apps/esm-implementer-tools-app/src/configuration/editable-value.component.tsx]: @brandones I hacked in the `as any` here to move the PR forward. By added the `lodash-es` types, TS started to complain here that `unset` returns a `boolean` which is not assignable to `Config`. Not sure if that indicates a bug or if that's the way it should be. Maybe this needs some extra attention though and I think that you know most about the config API, hence the tag.

Comment (brandones) [reply packages/apps/esm-implementer-tools-app/src/configuration/editable-value.component.tsx]: Oh shoot, thanks for the heads up!

**Pull request merged**